eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -59,6 +59,11 @@
     <Bug pattern="PZLA_PREFER_ZERO_LENGTH_ARRAYS"/>
   </Match>
 
+  <Match>
```

Comment:
Maybe we should add a reason for this spotbugs exclude? I thought our goal was to avoid adding excludes for track 2 library.

Reply:
I will add a comment for this and report an issue. This was added for the azure-core-test package.